sdk/healthdataaiservices/azure-health-deidentification/ci.yml

```diff
@@ -0,0 +1,41 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - release/*
```

Comment:
I had this error so if you copied this from me you might need to add `feature/*` here too.